pkg/upgrade/upgrader/jiva_upgrade.go

```diff
@@ -281,47 +282,61 @@ func getPVCDeploymentsNamespace(
 	return ns, nil
 }
 
-func validateSync(ctrlLabel, namespace string) error {
-	klog.Infof("Verifying replica sync")
-	quorum := false
+func getReplicationFactor(ctrlLabel, namespace string) (int, error) {
 	ctrlList, err := podClient.WithNamespace(namespace).List(
 		metav1.ListOptions{
 			LabelSelector: ctrlLabel,
 		})
 	if err != nil {
-		return err
+		return 0, err
 	}
 	if len(ctrlList.Items) == 0 {
-		return errors.Errorf("no deployments found for %s in %s", ctrlLabel, namespace)
+		return 0, errors.Errorf("no deployments found for %s in %s", ctrlLabel, namespace)
 	}
 	ctrlPod := ctrlList.Items[0]
+	return strconv.Atoi(ctrlPod.Spec.Containers[0].Env[0].Value)
```

Comment:
Add a comment over here that Env[0].Key is "REPLICATION_FACTOR"

Reply:
Done